iina/Formatters.swift

```diff
@@ -23,3 +23,45 @@ class DecimalFormatter : NumberFormatter {
     return true
   }
 }
+
+class RestrictedNumberFormatter : NumberFormatter {
+
+  init(_ min: Float?, max: Float?, isDecimal: Bool) {
+    super.init()
+    if isDecimal {
+      self.numberStyle = .decimal
+    }
+    if let minValue = min {
+      minimum = minValue as NSNumber?
```

Comment:
> Swift should bridge min to NSNumber automatically, right?

Swift will NOT convert it automatically. Xcode arise a error says
> Cannot assign value of type 'Double?' to type 'NSNumber?'

And it suggest me to add ` as NSNumber?`

But I found that I do not have to unwrap it explicitly. So I will change this piece of code into
`minimum = min as NSNumber?`
`maximum = max as NSNumber?`